api/src/main/java/org/pmiops/workbench/cohorts/CohortMaterializationService.java

```diff
@@ -223,6 +252,50 @@ private void addFilterOnConcepts(TableQuery tableQuery, Set<Long> conceptIds,
     }
   }
 
+  @VisibleForTesting
+  CdrQuery getCdrQuery(@Nullable CohortReview cohortReview, SearchRequest searchRequest,
+      @Nullable Set<Long> conceptIds, DataTableSpecification dataTableSpecification) {
+    CdrVersion cdrVersion = CdrVersionContext.getCdrVersion();
+    CdrQuery cdrQuery = new CdrQuery();
+    cdrQuery.setBigqueryDataset(cdrVersion.getBigqueryDataset());
+    cdrQuery.setBigqueryProject(cdrVersion.getBigqueryProject());
+    List<CohortStatus> statusFilter = dataTableSpecification.getStatusFilter();
+    if (statusFilter == null) {
+      statusFilter = NOT_EXCLUDED;
+    }
+    ParticipantCriteria criteria = getParticipantCriteria(statusFilter, cohortReview,
+        searchRequest);
+    if (criteria.getParticipantIdsToInclude() != null
+        && criteria.getParticipantIdsToInclude().isEmpty()) {
+      // There is no cohort review, or no participants matching the status filter;
```

Comment:
Using an empty list as a sentinel I think is a bit brittle - instead of `participantIdsToInclude` can you just check for null'ness of `cohortReview` on L268?

Reply:
So it's possible to have no cohort review, but it's also possible to have a cohort review that doesn't actually include anybody based off the status filter. This handles both of those.